index.md

```diff
@@ -1,4 +1,5 @@
- > This platform addressed common industrial challenges such as machine connectivity and on-premises systems integration. It aims to significantly accelerate future industrial IoT developments, shorten time to value and drive production efficiencies while addressing common industrial challenges. It was dissolved end of 2022. Some of the work is continued (see comments for different working groups results)
+ > This platform addressed common industrial challenges such as machine connectivity and on-premises systems integration. It aims to significantly accelerate future industrial IoT developments, shorten time to value and drive production efficiencies while addressing common industrial challenges.
+ The OMP was dissolved end of 2022. The work of the SDS working group is continued in the [Eclipse Semantic Modeling Framework](https://projects.eclipse.org/projects/dt.esmf/developer)
```

Comment:
Might be better to link the overview ("homepage"): https://projects.eclipse.org/projects/dt.esmf

Reply:
Ok, good point, since the overview page can also be edited later with relevant links. My initial thinking was to make it as easy as possible to get to all the other links.